### PR TITLE
Update the Default Voyager version from 10.0.0 to 12.0.0 

### PR DESCRIPTION
**Pull request by anpanigr**
As per Voyager Doc 
https://github.com/voyagermesh/voyager#supported-versions

Currently the GA version is 12.0.0  ( GA ) .  Need to update the sample utility script to make 12.0.0 as default.
Enhanced the script to check the creation of  CRD  `ingresses.voyager.appscode.com` beside the voyager operator pod creation. 

I noticed that the integration test uses 12.0.0 thru the variable TestConstants.java.VOYAGER_CHART_VERSION = "12.0.0"

### CODE REVIEW

**Review by rjeberhard — APPROVED** [review]

File: kubernetes/samples/charts/util/setupLoadBalancer.sh [hunk 1]
```diff
@@ -216,7 +245,8 @@ function createNginx() {
 
   if [ "$(helm list --namespace $NNAME | grep $NNAME |  wc -l)" = 0 ]; then
```
Comment (xiancao) [inline kubernetes/samples/charts/util/setupLoadBalancer.sh:246]: This should be:
if [ "$(helm list --namespace $NSPACE| grep $NNAME |  wc -l)" = 0 ]; then

File: kubernetes/samples/charts/util/setupLoadBalancer.sh [hunk 2]
```diff
@@ -228,7 +258,9 @@ function createNginx() {
   count=0
   tpod=$(kubectl get po -n ${NSPACE} --no-headers | awk '{print $1}')
   while test $count -lt $max; do
-    if test "$(kubectl get po -n ${NSPACE} --no-headers | awk '{print $2}')" = 1/1; then
+    status=$(kubectl get po -n ${NSPACE} --no-headers 2> /dev/null \
+                | awk '{print $2}')
```
Comment (xiancao) [inline kubernetes/samples/charts/util/setupLoadBalancer.sh:262]: You can put these in one line

File: kubernetes/samples/charts/util/setupLoadBalancer.sh [hunk 3]
```diff
@@ -264,7 +296,7 @@ function main() {
     usage
   fi
   if [ "$2" != traefik ] && [ "$2" != voyager ] && [ "$2" != nginx ]; then
-    echo "[ERROR] The second  parameter MUST be either traefik or voyager or inginx "
+    echo "[ERROR] The second parameter MUST be either traefik, voyager or inginx "
```
Comment (xiancao) [inline kubernetes/samples/charts/util/setupLoadBalancer.sh:299]: typo here:
inginx -> nginx

File: kubernetes/samples/charts/util/setupLoadBalancer.sh [hunk 4]
```diff
@@ -71,17 +72,43 @@ function createVoyager() {
   count=0
   vpod=$(kubectl get po -n ${VSPACE} --no-headers | awk '{print $1}')
   while test $count -lt $max; do
-    if test "$(kubectl get po -n ${VSPACE} --no-headers | awk '{print $2}')" = 1/1; then
+    status=$(kubectl get po -n ${VSPACE} --no-headers 2> /dev/null \
+                | awk '{print $2}')
```
Comment (xiancao) [inline kubernetes/samples/charts/util/setupLoadBalancer.sh:76]: You can put these in one line

File: kubernetes/samples/charts/util/setupLoadBalancer.sh [hunk 5]
```diff
@@ -71,17 +72,43 @@ function createVoyager() {
...
+  kubectl describe pod/${vpod} -n ${VSPACE}
+  helm list -n ${VSPACE}
+  
+  max=20
+  count=0
+  crd="ingresses.voyager.appscode.com"
+  echo "Checking availability of Custom Resource Definition [${crd}]"
+  while test $count -lt $max; do
+   obj=$(kubectl get crd ${crd} -n ${VSPACE} \
+          --no-headers 2>/dev/null | awk '{print $1}')
```
Comment (xiancao) [inline kubernetes/samples/charts/util/setupLoadBalancer.sh:98]: You can put these in one line

File: kubernetes/samples/charts/util/setupLoadBalancer.sh [hunk 6]
```diff
@@ -115,7 +142,9 @@ function createTraefik() {
   count=0
   tpod=$(kubectl get po -n ${TSPACE} --no-headers | awk '{print $1}')
   while test $count -lt $max; do
-    if test "$(kubectl get po -n ${TSPACE} --no-headers | awk '{print $2}')" = 1/1; then
+    status=$(kubectl get po -n ${TSPACE} --no-headers 2> /dev/null \
+                | awk '{print $2}')
```
Comment (xiancao) [inline kubernetes/samples/charts/util/setupLoadBalancer.sh:146]: you can put these in one line

File: kubernetes/samples/charts/util/setupLoadBalancer.sh [hunk 7]
```diff
@@ -228,7 +258,9 @@ function createNginx() {
   count=0
   tpod=$(kubectl get po -n ${NSPACE} --no-headers | awk '{print $1}')
   while test $count -lt $max; do
-    if test "$(kubectl get po -n ${NSPACE} --no-headers | awk '{print $2}')" = 1/1; then
+    status=$(kubectl get po -n ${NSPACE} --no-headers 2> /dev/null \
+                | awk '{print $2}')
+    if [ ${status} == "1/1" ]; then
       echo "Nginx operator pod is running now."
       kubectl get pod/${tpod} -n ${NSPACE}
       kubectl exec -it $tpod -n nginx -- /nginx-ingress-controller --version``
```
Comment (xiancao) [inline kubernetes/samples/charts/util/setupLoadBalancer.sh:266]: change this to:
kubectl exec -it $tpod -n ${NSPACE} -- /nginx-ingress-controller --version``

**Review by xiancao — CHANGES_REQUESTED** [review]

File: kubernetes/samples/charts/util/setupLoadBalancer.sh (hunk 4 above)
Comment (anpanigr) [reply kubernetes/samples/charts/util/setupLoadBalancer.sh]: done

File: kubernetes/samples/charts/util/setupLoadBalancer.sh (hunk 5 above)
Comment (anpanigr) [reply kubernetes/samples/charts/util/setupLoadBalancer.sh]: done

File: kubernetes/samples/charts/util/setupLoadBalancer.sh (hunk 6 above)
Comment (anpanigr) [reply kubernetes/samples/charts/util/setupLoadBalancer.sh]: done

File: kubernetes/samples/charts/util/setupLoadBalancer.sh (hunk 2 above)
Comment (anpanigr) [reply kubernetes/samples/charts/util/setupLoadBalancer.sh]: done

File: kubernetes/samples/charts/util/setupLoadBalancer.sh (hunk 7 above)
Comment (anpanigr) [reply kubernetes/samples/charts/util/setupLoadBalancer.sh]: done

File: kubernetes/samples/charts/util/setupLoadBalancer.sh (hunk 3 above)
Comment (anpanigr) [reply kubernetes/samples/charts/util/setupLoadBalancer.sh]: fixed the typo

File: kubernetes/samples/charts/util/setupLoadBalancer.sh (hunk 1 above)
Comment (anpanigr) [reply kubernetes/samples/charts/util/setupLoadBalancer.sh]: done

**Review by xiancao — APPROVED** [review]
LGTM

**Pull request merged**